src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Process/Process.ts

```diff
@@ -112,16 +114,18 @@ export var workflowSwitchDirective = (
             });
 
             scope.switchState = (newState) => {
-                if ( ! $window.confirm("Will switch to process state " + newState + ". (Page will reload)")) {
-                    return;
-                }
-                adhHttp.get(scope.path).then((process) => {
-                    process.data[SIWorkflow.nick] = {
-                        workflow_state: newState
-                    };
-                    process.data[SIName.nick] = undefined;
-                    adhHttp.put(scope.path, process).then((response) => {
-                        $window.parent.location.reload();
+                $q.all([$translate("TR__WILL_SWITCH_TO_STATE"), $translate("TR__WILL_RELOAD_PAGE")]).then((translated) => {
+                    if (!$window.confirm(translated[0] + newState + ". " + translated[1])) {
```

Comment:
This should use [interpolation](https://angular-translate.github.io/docs/#/guide/06_variable-replacement). But it is not really relevant or urgent.